db/models.py

```diff
@@ -50,3 +53,57 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE)
```

Comment:
Add `related_name` for FK (ForeignKey) fields here and in the `Ticket` model as well